sgrep.py

```diff
@@ -417,7 +463,7 @@ def validate_configs(configs: Dict[str, Any]) -> Tuple[Dict[str, Any], Dict[str,
             print_error(f"{config_id} is missing `{RULES_KEY}` as top-level key")
             errors[config_id] = config
             continue
-        rules = config.get(RULES_KEY, [])
+        rules = config.get(RULES_KEY)
```

Comment:
```suggestion
        rules = config.get(RULES_KEY, [])
```

Reply:
I explicitly want us to error here if it doesn't exist, because (1) I think it should be a failure to parse a yaml file with no rules defined (2) I just checked for RULE_KEY not in config above